conanfile.py

```diff
@@ -21,4 +21,8 @@ def package(self):
         self.copy("*.a", dst="lib", src="hello/lib")
 
     def package_info(self):
+        self.cpp_info.cxxflags = [
```

Comment:
But this will most likely fail in any other platform like Windows Visual Studio. 

This is not responsibility of every recipe, but this should be part of the build toolchain (the build scripts, or the generated files by Conan).

Moreover, this is a bit outdated, Conan docs should no longer point to my own personal repo, and it is moving instead to an "examples" repo that includes CI to validate these things.

I suggest moving the whole link and example there to the examples repo instead of here.